openscenario/openscenario_interpreter/src/syntax/dynamic_constraints.cpp

```diff
@@ -37,7 +38,8 @@ DynamicConstraints::DynamicConstraints(const pugi::xml_node & node, Scope & scop
   max_deceleration(readAttribute<Double>("maxDeceleration", node, scope, Double::infinity())),
   max_deceleration_rate(
     readAttribute<Double>("maxDecelerationRate", node, scope, Double::infinity())),
-  max_speed(readAttribute<Double>("maxSpeed", node, scope, Double::infinity()))
+  max_speed(readAttribute<Double>("maxSpeed", node, scope, Double::infinity())),
+  max_jerk(readAttribute<Double>("maxJerk", node, scope, Double::infinity()))
```

Comment:
maxJerk field does not exist in [DynamicConstraints.](https://www.asam.net/static_downloads/ASAM_OpenSCENARIO_V1.2.0_Model_Documentation/modelDocumentation/content/DynamicConstraints.html)
Please remove it.